Makefile

```diff
@@ -12,6 +13,7 @@ venv:
 
 .PHONY: setup
 setup:
+	python -m pip install --upgrade wheel
```

Comment:
Similarly wheel is not included by default with pip on a fresh install of Ubuntu 20.04 LTS.